script.py

```diff
@@ -256,14 +256,20 @@ def printLogo():
     )
 
 def main():
+    arguments = Arguments.parse()
+    GLOBAL.arguments = arguments
 
-    if not Path(GLOBAL.defaultConfigDirectory).is_dir():
-        os.makedirs(GLOBAL.defaultConfigDirectory)
-
-    if Path("config.json").exists():
-        GLOBAL.configDirectory = Path("config.json")
+    if arguments.config:
+        if arguments.use_local_config:
+            sys.exit()
+        GLOBAL.configDirectory = Path(arguments.config)
```

Comment:
There should be a validation that the given file actually exists.

Reply:
Added validation with log message.